src/test/java/io/supertokens/test/emailpassword/MultitenantEmailPasswordTest.java

```diff
@@ -0,0 +1,2 @@
+package io.supertokens.test.emailpassword;public class MultitenantEmailPasswordTest {
```

Comment:
format properly

Reply:
Done, but need to update the tests to use API instead of storage.

Reply:
Ok. Keeping this issue open